learning_journal_basic/learning_journal_basic/__init__.py

```diff
@@ -1,12 +1,14 @@
 from pyramid.config import Configurator
-
+import os
 
 def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
+    settings['sqlalchemy.url'] = os.environ['BLOGENTRY_DB_URL']
```

Comment:
You have a environment variable for the same thing in your initdb script with a different name.